### ISSUE TITLE
CR1000 timing

### ISSUE THREAD

**Issue by USGSminc** [SacArea-BayDeltaProjects/HydroRDITemplate#55]
The CR1000s are too slow!

To do list:
-Make a variable for the burst delay timer. Waiting until maxburstcnt+8 is fine for the CR6 but it is too close for a CR1000. On CM72 I used maxbustcnt+15 and we were getting scan end at about 55seconds. With the +8 it was 62 and it was skipping.

-Currently, we are taking in all the ADVM data, then doing burst stuff, then doing the ADVM decodes. I think moving all the ADVM code together and do the burst after will work better. That way decodes occurs while we are waiting to start the bust. If we do that as soon as the burst is done there are no long processes occurring afterwards that could cause a skipped scan. If decodes takes extra time that may push back the burst a little bit. But I prefer that over a skipped scan.

**Comment by USGSminc** [comment]
V4.24 incorporates the first to do list item.
Second item needs more testing before being implemented.

**Comment by violetteta8** [comment]
While I'm working on the SUNA bug, i'll start to look at rearranging when decoding happens.

**Comment by USGSminc** [comment]
V 4.41 moves the water quality until after the entire ADVM process. Also altered the CR1000 delay to 30 so basically it will start the burst as soon as it can. At 15 it was finishing the burst was finishing around 52 seconds. Then decodes would run and push it towards 55-57 seconds but occasionally it would get dangerously close to skipping a scan. In theory, now decodes would finish while we were delaying to start the burst so as soon as the burst was done the scan would be complete and we wouldn't skip. But just in case, I upped it to 30, making the burst start as soon as it can. I prefer not having exact control over the start time in exchange for not missing the scan that allows us to write the Aquarius table. I kept the CR6 the same so on that platform we can push the WQ sample until right before the 15min punch. If we find the alteration in decode placement to guarantee us finishing a scan on a CR1000 with time to spare we can bring back the delay to better control the start time,
